libcxx/include/__config

```diff
@@ -421,7 +421,7 @@ _LIBCPP_HARDENING_MODE_DEBUG
 #  if !defined(_LIBCPP_ENABLE_EXPERIMENTAL) && !defined(_LIBCPP_BUILDING_LIBRARY)
 #    define _LIBCPP_HAS_NO_INCOMPLETE_PSTL
```

Comment:
@philnik777 @ldionne should we rename PSTL too?

Reply:
I'll let Louis decide. He's currently working on some PSTL refactoring, so maybe not right now.